bedevere/bpo.py

```diff
@@ -154,3 +173,12 @@ def create_hyperlink_in_comment_body(body):
             leftover_body = leftover_body[presence:]
     new_body = new_body + leftover_body
     return new_body
+
+
+async def _validate_issue_number(issue_number):
+    """Make sure the issue number exists on bugs.python.org."""
+    url = f"https://bugs.python.org/issue{issue_number}"
+    async with aiohttp.ClientSession() as session:
```

Comment:
Instead of creating a new session for `aiohttp`, you could pass it in through the `__main__` method where dispatch occurs to try and maximize connection pooling. All routes should be acceptings `*args, **kwargs` so passing in a `session` keyword argument should work fine.